app/lib/package/models.dart

```diff
@@ -102,7 +102,9 @@ class Package extends db.ExpandoModel<String> {
 
   /// Set to `true` if package should not be advertised on the front page,
   /// may otherwise be `false`.
-  @db.BoolProperty(required: true)
+  ///
+  /// TODO: remove after `2020.10.09` is no longer an accepted runtimeVersion.
+  @db.BoolProperty()
```

Comment:
Nice transition to `required: false` :D

Can we add `@deprecated` ?

Reply:
`@deprecated` doesn't seem to be necessary for now, I'd rather aim for removing it soon :).